src/sparseml/transformers/export.py

```diff
@@ -304,6 +362,11 @@ def export(
     )
     _LOGGER.info(f"Model exported to: {onnx_path}")
 
+    deployment_folder_dir = create_deployment_folder(
```

Comment:
This violates to some extent the Single Responsibility Principle (SRP), but this choice was better than the alternatives (creating  a separate entry point for deployment folder creation, adding additional flags to the export function etc).

Reply:
would disagree on this point - export's purpose is to get an artifact ready for deployment, this is now defined as a directory that includes both an onnx export and config files